src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspace.cs

```diff
@@ -72,6 +72,8 @@ protected override void OnDocumentClosing(DocumentId documentId)
             }
         }
 
+        internal override bool IgnoreUnchangeableDocumentsWhenApplyingChanges => true;
```

Comment:
Do we have a bug for discussing whether this should be the default? It seems like if all our workspaces are doing this should we just change the default, but I can 100% support not wanting to do that right now for risk reduction given where we are in the preview cycle.

Reply:
Sure, I can file an issue for this. However, I'm still hoping that we can get rid of this whole unchangeable document thing sooner rather than later...